src/main/java/org/primeframework/mvc/sampleapp/action/LoginAction.java

```diff
@@ -0,0 +1,29 @@
+package org.primeframework.mvc.sampleapp.action;
+
+import com.google.inject.Inject;
+import org.primeframework.mvc.action.annotation.Action;
+import org.primeframework.mvc.action.result.annotation.ReexecuteSavedRequest;
+import org.primeframework.mvc.sampleapp.session.OurUser;
+import org.primeframework.mvc.security.UserLoginSecurityContext;
+
+import java.util.UUID;
+
+@Action
+@ReexecuteSavedRequest(uri = "/")
+public class LoginAction extends BaseAction {
+    @Inject
+    private UserLoginSecurityContext userLoginSecurityContext;
```

Comment:
Since this is for example purposes, I think this should be a a private final field, and we should use constructor injection. Same for other actions as well.

Reply:
Done